Assets/Scripts/Menu Scripts/SensorManager.cs

```diff
@@ -36,6 +31,7 @@ void Awake()
     public void initializeSensorUI(List<ROSSensorConnectionInterface> allSensors)
     {
         sensorList.Clear();
+        deleteOldSensors();
```

Comment:
Do we need to delete the buttons and re create them?
We can just change the name of the sensors and remove/add as needed too?
But we don't have to do this.